reclaimer/sounds/Encoder.py

```diff
@@ -0,0 +1,221 @@
+import ctypes, sys
+
+try:
+    from pyogg.vorbis import *
+    from pyogg.ogg import ogg_page_eos
+except ImportError:
+    input("Unable to find pyOGG. \nUse 'pip install pyogg' via cmd on Windows, or 'pip3 install pyogg' via terminal for linux")
+
+#seems to be a bug with pyogg defintion. Redefine to get around.
+libvorbis.vorbis_bitrate_flushpacket.argtypes = [vd_p, op_p]
+def vorbis_bitrate_flushpacket(vd, op):
+        return libvorbis.vorbis_bitrate_flushpacket(vd, op)
+
+"""
+    OGG Encoder:
+        Creation: 06/15/2020
+        Last Edit: 06/18/2020 (Zatarita)
+
+    KNOWN ISSUES:
+        Channels above stereo get mixed. (I don't have a multi-channel speaker setup so I cant fix)
+
+    To Do:
+        for the love of all things holy, thread this.
+"""
+
+class OGGEncoder():
+    vorbisinfo = vorbis_info()                                      #Vorbis encoding info
+    vorbiscomment = vorbis_comment()                                #Vorbis comment struct
+    vorbisencoderstate = vorbis_dsp_state()                         #Encoder state for one instance
+    vorbisblock = vorbis_block()                                    #Data for a block of audio
+
+    packet = ogg_packet()                                           #Raw packet of data
+    packetcomment = ogg_packet()                                    #Raw packet of data for comments
+    packetcode = ogg_packet()                                       #Raw packet of data for code
+    page = ogg_page()                                               #Fundamental unit of framing and interleave
+    streamstate = ogg_stream_state()                                #Handles current encode/decode state
+
+    Output = []
+
+    def __init__(self, samples, channelcount, samplerate, quality, samplecount, log = False):
+        self.channel_count = channelcount                            #Amount of audio channels (mono, l/r, ect)
+        self.sample_rate = samplerate                                #How many samples per second
+        self.sample_rate = quality                                      #Quality of the OGG compressison
+        self.float_samples = samples                                 #16 bit pulse code modulation data (wave)
+        self.sample_count = samplecount
+        self.encoder_version = "0.1.0"                               #Encoder Version
+
+        self.log = log                                              #For debugging
+
+        self.initializeinterpreter()                                #Initialize the interpreter
+        self.encode()                                               #Do the thing
+        self.releasevorbis()                                        #Cleanup
```

Comment:
I'd like it if we could move the comments to be above what line they're describing. Right now the way they're put here they take up a lot of horizontal space which gets in the way when editing multiple files at once. It also makes it so that developers have to actively try to keep the spacing proper.